tests/networkpolicy_test.go

```diff
@@ -104,14 +90,14 @@ var _ = Describe("[rfe_id:150][crit:high][vendor:cnv-qe@redhat.com][level:compon
 
 		It("[test_id:1511] should be failed to reach vmia from vmib", func() {
 			By("Connect vmia from vmib")
-			ip := vmia.Status.Interfaces[0].IP
-			assertPingFail(ip, vmib)
+			Expect(vmia.Status.Interfaces[0].IPs).NotTo(BeEmpty())
```

Comment:
Move this expectation with `ExpectWithOffset(2` into the `pingEventually`

Reply:
`assertPingFailBetweenVMs` does `for _, ip := range vmidst.Status.Interfaces[0].IPs`
before it call `pingEventually`, so it cant be part of `pingEventually`,

Moroever, i think we even shouldnt move it into assertPingFailBetweenVMs (or its counter part) success
because its a preliminary check that the test is doing.

Same answer for all the tests because they have same comment.

Reply:
Well test is testing ping from vmiab to vmia, so I don't expect to see explicitly a checking on IPs, I would do that as part of `pingEventually` with the benefict of not repeating that at all the test.

Reply:
pingEventually test just one ip at, the for is above it, so at least lets do it one level above?
another thing is that first will work on the change Alon and Miguel asked, and then we even might remove the eventually, and so on from the ping (but can adapt this change in the new structure)

Reply:
Done (for all 7 tests)